setup.py

```diff
@@ -94,6 +94,7 @@
         "tensorflow>=2.4,<=2.12.1",
         "transformers>=4.36,<4.38",
     ],
+    "onnx": "optimum-onnx @ git+https://github.com/huggingface/optimum-onnx.git@mv-onnx",
```

Comment:
the exporters and onnxruntime(-gpu) will also need to install optimum-onnx

Reply:
good point added in https://github.com/huggingface/optimum/pull/2298/commits/8806db59850fda63d4a5e5aaaf12d4d64bf54db8